docs/source/science/OPERA/OPERA_Surface_Displacement.ipynb

```diff
@@ -0,0 +1,5515 @@
+{
```

Comment:
<p>The plot is missing in the rendered notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>